### ISSUE TITLE
A11y_VisualStudioCodeClient_Version Control_ Screen reader:NVDA/Narrator is not announcing comments like “You last week add first commit” in visual code editor

### ISSUE THREAD

**Issue by VijayaBharathyA** [microsoft/vscode#159543]
### GitHub Tags:
#A11yMAS; #A11yTCS; #A11ySev2; #DesktopApp; #Win32; #Visual Studio Code Client; #BM-VisualStudioCodeClient-Win32-Aug2022; #WCAG1.3.1; #NVDA; #ScreenReader; #Narrator;

### Environment and OS details:
Visual Studio Code Version: 1.70.2 (user setup)
Windows 11 Pro version 21H2 (OS Build 22000.856)

### Repro Steps:

1.	Open Visual studio code editor and create branch in new folder by using https://code.visualstudio.com/docs/editor/versioncontrol
2.	Open a file in that folder
3.	Navigate in file using keyboard
4.	Verify that screen reader announcing the comments like “You last week add first commit” 
Or not 

### Actual Result:
NVDA/Narrator is not announcing comments like “You last week add first commit” in visual code editor

### Expected Result:
NVDA/Narrator should announce comments like “You last week add first commit” in visual code editor

### Attachment:
![A11y-VisualStudioCodeClient-Vers](https://user-images.githubusercontent.com/100189407/187403263-0075817f-7d81-4407-9c09-2d54e8079b20.gif)
![A11y-VisualStudioCodeClient-Vers (1)](https://user-images.githubusercontent.com/100189407/187406400-b6b36f75-7d3c-48fc-bf2c-a52531e54630.gif)

**Comment by isidorn** [comment]
@VijayaBharathyA thanks for filling this.
@lszomoru I think this issue makes sense. So the title of this button should also contain the branch name
Code pointer https://github.com/microsoft/vscode/blob/main/extensions/git/src/statusbar.ts#L32

Similar to how you compute the title. Let me know what you think.

**Comment by VijayaBharathyA** [comment]
#Closed;
Issue not repro we have alternate tooltip there that is accessible using keyboard first press Ctrl + K, and then Ctrl + I. Screen reader also announcing that hence closing  the bug
![159543](https://user-images.githubusercontent.com/100189407/220868458-074e43fe-6fdd-40e5-83eb-30a665d0f58b.png)
![159543_NVDA](https://user-images.githubusercontent.com/100189407/220868466-3ed63962-b642-4b25-b038-0a99ec3db8c5.png)

**Comment by VijayaBharathyA** [comment]
### Verified issue in below Environment:
Version: 1.76.0-insider (user setup)
Commit: d0acdbff1f30eac7cc68930013485276d1bae0ec
Date: 2023-02-23T05:23:26.617Z
Electron: 19.1.10
Chromium: 102.0.5005.196
Node.js: 16.14.2
V8: 10.2.154.26-electron.0
OS: Windows_NT x64 10.0.22621
Sandboxed: Yes
Issue is fixed
